backend/entities/dataset_files/dataset_file.py

```diff
@@ -41,12 +40,18 @@ class DatasetFile:
     """
     df: pd.DataFrame
     categories: set
-    bias_calculator: BiasCalculator
```

Comment:
Again good idea for removing bias calculator from a class that's supposed to contain just data